viewpoint-api/db.txt

```diff
@@ -5,8 +5,10 @@ use viewpoint;
 
 create table users
 (
-id_usu int primary key auto_increment,
-email varchar(60) not null unique,
+id_user int primary key auto_increment,
+email_user varchar(80) not null unique,
```

Comment:
Hoje, um e-mail pode ter até 254 caracteres, esse campo não atende.

Fonte: http://stackoverflow.com/questions/386294/what-is-the-maximum-length-of-a-valid-email-address